packages/media/src/getFileExtFromUrl.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const getFileExtFromUrl = (url) =>
+  url && url.substring(url.lastIndexOf('/')).split('.').slice(1, 2).join('.');
+
+export default getFileExtFromUrl;
```

Comment:
```suggestion
/**
 * Returns the file extension from a given URL.
 *
 * @param {?string} url File URL.
 * @return {?string} File extension.
 */
const getFileExtFromUrl = (url) =>
  url?.substring(url?.lastIndexOf('/')).split('.').slice(1, 2).join('.');
```

Reply:
Does this work?

Reply:
Yes. I tested this locally before sharing. Feel free to test it as well.